### DISCUSSION TITLE
Newbie to Cuda-Q

### DISCUSSION THREAD

**Discussion by franciscofrade870** [NVIDIA/cuda-quantum#1984] [Q&A]
Whats good guys, as the header states Im a complete newbie to all of this. Im more of starting this discussion for my girl. We're both in the cyber security field but we contribute in two different areas of the field. Long story short she's wanting to do a research project that involves cryptography and quantum computing ( apologies for the misspelling) combining the two together. I've helped her so far to finding research material, going over fundamentals, quantum walks, entanglements, generation, distribution, etc. My question is, can this program allow her to test her research question? And is this program considered a quantum simulator? Any and all information would help.

**Comment by efratshabtai** [comment]
CUDA-Q is a platform for accelerated quantum computing. It has a programming model that can be used to write quantum applications and algorithms. These programs can run on a variety of quantum simulators (GPU accelerated) and QPUs. See an overview here: https://developer.nvidia.com/cuda-q.
Please look into our documentation to learn more about CUDA-Q: https://nvidia.github.io/cuda-quantum/latest/index.html.
Once you start to evaluate it, you can post specific questions here or open issues.
